ckanext/datajson/tests/datajson-samples/large-spatial.data.json

```diff
@@ -1,8 +1,8 @@
 {
   "@type": "dcat:Catalog",
-  "describedBy": "https://project-open-data.cio.gov/v1.1/schema/catalog.json",
-  "conformsTo": "https://project-open-data.cio.gov/v1.1/schema",
-  "@context": "https://project-open-data.cio.gov/v1.1/schema/catalog.jsonld",
+  "describedBy": "https://resources.data.gov/schemas/dcat-us/v1.1/schema/catalog.json",
+  "conformsTo": "https://resources.data.gov/schemas/dcat-us/v1.1/schema",
+  "@context": "https://resources.data.gov/schemas/dcat-us/v1.1/schema/catalog.jsonld",
```

Comment:
Don't forget that the data.json files we consume from agencies are still going to have the old https://project-open-data.cio.gov/v1.1/schema identifier, so we want to make sure our test fixtures reflect that. I don't think we should be editing any of these test samples.